### ISSUE TITLE
Does not build with latest gradle

### ISSUE THREAD

**Issue by alxndrsn** [commonsguy/cwac-wakeful#23]
```
$ gradle clean

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/user/workspaces/medic/cwac-wakeful/demo/build.gradle' line: 1

* What went wrong:
A problem occurred evaluating project ':demo'.
> Failed to apply plugin [id 'com.android.application']
   > Gradle version 2.2 is required. Current version is 2.12. If using the gradle wrapper, try editing the distributionUrl in /Users/user/workspaces/medic/cwac-wakeful/gradle/wrapper/gradle-wrapper.properties to gradle-2.2-all.zip

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 2.997 secs
```

This looks like a bug in android build tools, but may actually be caused because the `gradlew` bin files are not included in the repo, which meant I was running a newer version of gradle.

**Comment by commonsguy** [comment]
Addressed in v1.1.0 -- thanks!

**Comment by alxndrsn** [comment]
That was quick – thank you also!
